.github/workflows/lint.yml

```
@@ -0,0 +1,21 @@
+name: Lint ModuleSite Code
+
+# Trigger the workflow on all pull requests
+on: pull_request
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout PR
+        uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '14'
```

Should use a newer Node.js version, the build script uses 16.
```suggestion
          node-version: '16'
```